apps/projects/migrations/0006_initialize_insights.py

```diff
@@ -16,7 +16,6 @@ def initialize_insights(apps, schema_editor):
     Like = apps.get_model("a4_candy_interactive_events", "Like")
     LiveQuestion = apps.get_model("a4_candy_interactive_events", "LiveQuestion")
     MapIdea = apps.get_model("a4_candy_mapideas", "MapIdea")
-    Topic = apps.get_model("a4_candy_topicprio", "Topic")
```

Comment:
what was the issue with the `Topic`? I'd like to understand.

Reply:
it's just that `Topic` is not used, because topics are created by initiators and we are only interested in comments on topics and not topics themselves. The comments of topics are identified by `model_name == "topic"` below.